### ISSUE TITLE
Portugal - more channels needed from tvepg.eu

### ISSUE THREAD

**Issue by Robinmcgvr** [Robinmcgvr/Kapow-Data#193]
Cinemundo (to 06/03/2023)
Fox (to 06/03/2023)
Sic K (to 14/02/2023)
Sic Mulher (14/02/2023)
 
Need adding to this source and back dated 

Matteo emailed

**Comment by Robinmcgvr** [comment]
Matteo has been unable to get back data for the above channels as the site is blocking him , he is looking into work arounds , but for now we only have data from 09/07 onwards

**Comment by Robinmcgvr** [comment]
We do have sic mulher and Fox on guia-tv.pt

**Comment by RichardFord** [comment]
> Matteo has been unable to get back data for the above channels as the site is blocking him , he is looking into work arounds , but for now we only have data from 09/07 onwards

Is he using the VPN? Ages ago we subscribed to a VPN for these purposes.

**Comment by Robinmcgvr** [comment]
I presume so as we normally manage - but his actual wording was
'The site is protecting the pages with captchas and at the moment I have not been able to recover the requested channels.
I am putting this on hold. 
As captchas are becoming more prevalent on websites, I'm going to study how they work.
There may be a chance to bypass them, at least for the simpler ones.'

**Comment by Robinmcgvr** [comment]
i presume he is talking about the 'I am not a robot' thing, which this site gest quite strict about from time to - even when I or others attempt to access the website directly

**Comment by RichardFord** [comment]
Yeah, bypassing a captcha is tricky maybe it will have to be done manually. No idea to be honest.
